src/components/LanguageSettings.js

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useState } from "react";
+import danSelected from "./icons/dan-selected.png";
+import danDeselected from "./icons/dan-deselected.png";
+import engSelected from "./icons/eng-selected.png";
+import engDeselected from "./icons/eng-deselected.png";
+
+import strings from "../i18n/definitions";
+
+import styled from "styled-components";
+
+const Option = styled.input`
+  opacity: 0;
+  cursor: pointer;
+  height: 0;
+  width: 0;
+`;
+
+const Icon = styled.img`
+  width: 30px;
+  height: auto;
+  margin-right: 4px;
+`;
+
+const Container = styled.label`
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+`;
+
+export default ({ useForceUpdate }) => {
```

Comment:
`export default ({ useForceUpdate }) => {`
Not naming the functions makes it super hard for inexperienced programmers to understand/maintain the code. (I know default exports can be named on import, but...)

Reply:
I will reluctantly take your point.